build/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM quay.io/app-sre/boilerplate:image-v0.4.0 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS builder
```

Comment:
Just noticed this. We can get rid of this second `FROM` line. (The boilerplate image is based on golang-1.13)

Reply:
done